### ISSUE TITLE
Ceph Dashboard Main Page Internal Server Error 500

### ISSUE THREAD

**Issue by JessedeJonge** [rook/rook#5500]
**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
After logging in, the main "Dashboard" page is empty and an 

`Internal Server Error, The server encountered an unexpected condition which prevented it from fulfilling the request.` error is shown. All the other pages like 'Cluster, Pools' etc work fine.

**Expected behavior:**

**How to reproduce it (minimal and precise):**

Deploy a cluster with ceph v15.2 and the following dashboard settings in the cluster yaml:
```
  dashboard:
    enabled: true
    urlPrefix: /storage
```

* Mgr logs
```
debug 2020-05-18T11:13:24.795+0000 7fd682451700  0 [dashboard ERROR request] [10.42.3.0:35450] [GET] [500] [0.099s] [admin] [513.0B] /storage/api/health/minimal
debug 2020-05-18T11:13:24.795+0000 7fd682451700  0 [dashboard ERROR request] [b'{"status": "500 Internal Server Error", "detail": "The server encountered an unexpected condition which prevented it from fulfilling the request.", "request_id": "49345af2-772f-47e4-b503-f6fdddbeb6d4"}
```

**Environment**:
* Rook version :
rook: v1.3.3
go: go1.13.8
* Storage backend version:
ceph/ceph:v15.2
* Storage backend status 
health: HEALTH_OK

**Comment by JessedeJonge** [comment]
Same issue as #5223, [This solves it for now.](https://github.com/rook/rook/issues/3106#issuecomment-489201188)

**Comment by lbogdan** [comment]
FYI: This was fixed with the latest `ceph/ceph:v15.2.4` image.
